src/services/storageService.ts

```diff
@@ -1,35 +1,28 @@
-import IConfigFetcher from "../interfaces/configFetcherInterface";
-import ILogSink from "../interfaces/logSinkInterface";
+import {IConfig} from "../interfaces/config";
 import { Log } from "../types";
 
 class StorageService {
-    configFetcher ?: IConfigFetcher|null = null;
-    logSink ?: ILogSink|null = null;
+    config?: IConfig | null = null;
```

Comment:
Storage service should keep source and sink implementation as its members. Not keep config directly